internal/storage/rocksdb/store.go

```diff
@@ -161,6 +196,20 @@ func (rdb *rocksDB) Close() error {
 	return nil
 }
 
+func (rdb *rocksDB) PutTTL(key []byte, value []byte, expireTS int64) error {
+	defer rdb.opts.statsCli.Timing("rocksdb.put.latency.ms", time.Now())
+	if expireTS > 0 {
+		b := make([]byte, tsLength)
+		binary.LittleEndian.PutUint64(b, uint64(expireTS))
+		value = append(value, b...)
+	}
+	err := rdb.db.Put(rdb.opts.writeOpts, key, value)
+	if err != nil {
+		rdb.opts.statsCli.Incr("rocksdb.put.errors", 1)
```

Comment:
Should we distinguish the metric name between TTL based and non-TTL based updates ?

Reply:
Done